ui/src/router/components/form/components/router_config/RulesPanel.js

```diff
@@ -13,17 +19,26 @@ import { get } from "../../../../../components/form/utils";
 import { RulesPanelFlyout } from "./RulesPanelFlyout";
 import { RuleCard } from "./rule_card/RuleCard";
 import { useOnChangeHandler } from "../../../../../components/form/hooks/useOnChangeHandler";
-import { newRule } from "../../../../../services/router/TuringRouter";
+import { newDefaultRule, newRule } from "../../../../../services/router/TuringRouter";
 
-export const RulesPanel = ({ rules, routes, onChangeHandler, errors = {} }) => {
+export const RulesPanel = ({ default_traffic_rule, rules, routes, onChangeHandler, errors = {}, default_traffic_rule_errors = {} }) => {
   const { onChange } = useOnChangeHandler(onChangeHandler);
 
   const onAddRule = () => {
+    // Default rule should only be added when there are custom rules
+    if (rules.length === 0) {
```

Comment:
Should this be `rules.length > 0 && ! default_traffic_rule` ?

Reply:
Updated the conditions!